pages/version/activity/view/index.tsx

```diff
@@ -50,6 +50,19 @@ const textWithIcon = (status: AppVersionEvent['status'], message: string) => {
   );
 };
 
+const downloadBuildLogButton = (
```

Comment:
Please use `PascalCase` for the component names

```suggestion
const DownloadBuildLogButton = (
```